.gitattributes

```diff
@@ -4,3 +4,14 @@ tests/data/*-win.php eol=crlf
 
 # Don't show bundled libraries as source files in GitHub
 bundle/**/* linguist-vendored
+
+/.github export-ignore
+/tests export-ignore
+/features export-ignore
+/bin export-ignore
+/phpunit.xml.dist export-ignore
+/composer.json export-ignore
```

Comment:
```suggestion
```

There's an argument in favour of keeping `composer.json` so people can quickly read it to better understand the package.

https://github.com/WordPress/plugin-check/issues/829
> You should also keep and/or link configuration files, as for example, the composer.json file in order to allow others to review, study, and yes, fork this code.

https://developer.wordpress.org/plugins/wordpress-org/common-issues/#included-unneeded-folders